dev/download.jl

```diff
@@ -0,0 +1,89 @@
+using Dates
+using TimeZones: DEPS_DIR
+using TimeZones.TZData: LATEST_FILE, LATEST_DELAY, latest_version, tzdata_version_dir, set_latest
+
+if Sys.iswindows()
+    if isdefined(Base, :LIBEXECDIR)
+        const exe7z = joinpath(Sys.BINDIR, Base.LIBEXECDIR, "7z.exe")
+    else
+        const exe7z = joinpath(Sys.BINDIR, "7z.exe")
+    end
+end
```

Comment:
You can just use the `exe7z` as defined in `TimeZones.TZData` instead of re-implementing it here

Reply:
I thought I'd remove it from `TimeZones.TZData` because it's not needed when it's using Artifacts, but if you want to have also the nonartifact support for longer version, then I'll put it back and use it from there.

Reply:
Everything returned back on it's place, using `extract` and `tzdata_download` from the package.